### ISSUE TITLE
Big Sur 11.6.2 not showing up

### ISSUE THREAD

**Issue by RobertoCMCosta** [dortania/OpenCore-Legacy-Patcher#813]
### Machine Model

iMac11,2

### Application Version

Latest Release

### Application Variant

TUI (Text User Interface)

### Have you verified whether this issue has been opened before? If no, your issue will be closed

true

### Have you verified whether this issue is covered in our guide? ie. Troubleshooting and Legacy Acceleration pages

true

### Is this issue with a 3rd party application?

false

### What versions of macOS are you seeing the problem on?

macOS 11, Big Sur

### Where does this issue happen?

Other/Non-Applicable

### What is the Issue?

I have a doubt and an issue.
Yesterday I did a clean install of MacOS 11.6.1 on a iMac 11,2 with upgraded metal gpu (Nvidia K1100M). I did this using the released 0.3.2 with  default settings. I chose Big Sur to avoid patching the root volume for now.
To my surprise everything worked out of the box: bluetooth, graphics accel., wifi. That was great for a 11 year old machine.
My doubt: The default settings disable SIP and secureboot. So I tried building OCPL again enabling them and installing the new EFI to an external disk for testing.  
So I reset PRAM and chose the external EFI holding option at startup. The iMac will either hang até the boot picker or show a black screen after hitting enter.
Is it possible to run this version of macOS on my iMac with SIP and secureboot on?

My issue: The newly released 11.6.2 version is not showing up at software update, although Safari 15.2 has shown up and been upgraded normally.

Thank you

### Any Additional Information

my specs

iMac 11,2 21,5 inches
i3 3,06 ghz
12gb ram
gpu nvidia k1100m
ssd sandisk

**Comment by khronokernel** [comment]
Should be resolved with 0.3.3

**Comment by RobertoCMCosta** [comment]
Hi, 
Thank you for the answer. 
I upgraded to 0.3.3 and after resetting pram and rebooting, software update found both macOS 12.1 and 11.6.2. However, installing 11.6.2 doesn't work, I mean, I've tried twice, it downloads, proceeds to installation, reboots and continues in 11.6.1.
Any clue? Could it be related to SIP and secure boot disabled? 
Should I download the full installer and update from an external drive?
